src/perf/simulations/launchAnalysis.scala

```diff
@@ -18,16 +15,41 @@ class launchAnalysis extends Simulation {
 	//	.extraInfoExtractor(extraInfo => List(extraInfo.response)) //for when we want to extract additional info for the simulation.log
 
 	val headers = Map("Authorization" -> s"Bearer ${accessToken}",
-						"Content-Type" -> "application/json") 
+						"Content-Type" -> "application/json")
 
 	val submissionBody = """{"methodConfigurationNamespace":"alex_methods","methodConfigurationName":"cancer_exome_pipeline_v2","entityType":"pair_set","entityName":"pair_set_1","expression":"this.pairs"}"""
 
+	// var b/c I don't know how to do exec() callbacks
+	var workspaceSubmissionsMap: Map[String, String] = Map()
+
 	val scn = scenario(s"launchAnalysis_${numUsers}")
 		.feed(tsv("{LIST_OF_WORKSPACE_NAMES}")) //feed the list of clones created for this test
 		.exec(http("submission_request")
 			.post("/api/workspaces/broad-dsde-dev/${workspaceName}/submissions")
 			.headers(headers)
-			.body(StringBody(submissionBody))) //launch the same submission in each workspace
+			.body(StringBody(submissionBody)) 	//launch the same submission in each workspace
+			.check(jsonPath("$.submissionId").saveAs("submissionId"))
+			)
+		.exec(session => {
+			val attrs = session.attributes
+			workspaceSubmissionsMap += (attrs.get("workspaceName").get.asInstanceOf[String] -> attrs.get("submissionId").get.asInstanceOf[String])
+			session
+		})
+	.doIf(session => workspaceSubmissionsMap.size == numUsers) {
+		exec(session => {
+			val randInt = scala.util.Random.nextInt
+			val filename = s"../user-files/data/launchAnalysis_${randInt}.tsv"
```

Comment:
I can do that, but it will still end up being an arbitrary, based on whichever task completed last: for example, the ID for submission 82 out of the run of 100.